cuda_bindings/cuda/bindings/nvvm.pyx

```diff
@@ -29,6 +29,7 @@ class Result(_IntEnum):
     ERROR_NO_MODULE_IN_PROGRAM = NVVM_ERROR_NO_MODULE_IN_PROGRAM
     ERROR_COMPILATION = NVVM_ERROR_COMPILATION
     ERROR_CANCELLED = NVVM_ERROR_CANCELLED
+    _P_NVVMINTERNAL_LOADING_ERROR = _NVVMRESULT_INTERNAL_LOADING_ERROR
```

Comment:
Do we want to expose this? (same question for cuda_bindings/cuda/bindings/nvjitlink.pyx)

Reply:
Interesting question. Let me ping you in the internal MR.

Reply:
addressed in commit 51faea7